samples/cellular/modem_shell/README.rst

```diff
@@ -702,6 +702,34 @@ Examples
 
 ----
 
+NTN
+===
+
+MoSh command: ``ntn``
+
+You can use the command for NTN helper functionality provided by the :ref:`lib_ntn` library.
+The command provides support for setting and invalidating the location to the modem, and enabling and disabling modem location updates from an external GNSS.
```

Comment:
What about internal GNSS? It's mentioned once in ntn.rst. I think it needs more attention w.r.t. what has to be done for it to work or keep it out of scope with clear phrase.

Reply:
Currently, it would have to be implemented by the application completely. This is something I think the library could have functionality for later. Internal GNSS is mentioned in the library documentation, but nowhere else. You can use it, but it's up to you to implement it. The API doesn't care where the location comes from.

It's also possible to do that with mosh commands, but I don't know if we should document it right now. I'm a bit reluctant to put the AT command sequence there now, because I'm not sure if there can still be changes.